jsr352/core/src/main/java/org/hibernate/search/jsr352/massindexing/MassIndexingJob.java

```diff
@@ -268,32 +280,32 @@ public ParametersBuilder rowsPerPartition(int rowsPerPartition) {
 		 * Build the parameters.
 		 *
 		 * @return The parameters.
+		 *
```

Comment:
> About the cleanup: it's nice (especially the utils stuff), and please leave it in your PR, we will merge it.

Thanks Yoann. My initial idea is to improve the code quality because the source codes in module JSR 352 are never review via PR, they are pushed directly into master branch. So I guess it would be a good idea to do some cleanup before merging into the official repo.

Some of the changes are made by IDE (IntelliJ):

- adding an empty line between different group of `import`
- adding an empty line between 2 parameters in the javadoc
- adding a new `\t` in each line of code inside `<pre>` tag in the javadoc

However, other changes have been added by myself:

- add a new line between last method and the last right embrace `}` of the class
- delete the first empty line of each method
- code refactoring

> But just so you know: in general we avoid this kind of formatting changes (like the empty line above) that have little value, and mess with the commit history without a good reason.
> This is not a hard rule, [...]

Yes, you're right. I'll try to avoid this the next time. And create a new JIRA ticket for this kind of cleanup.